packages/grafana-toolkit/src/plugins/utils.ts

```diff
@@ -98,7 +98,7 @@ export async function readGitLog(): Promise<GitLogInfo | undefined> {
     let exe = await execa('git', [
       'log',
       '-1', // last line
-      '--pretty=format:{%n  "commit": "%H",%n  "tree": "%T",%n  "subject": "%s",%n  "author": {%n    "name": "%aN",%n    "email": "%aE",%n    "time":"%at"  },%n  "commiter": {%n    "name": "%cN",%n    "email": "%cE",%n    "time":"%ct"  }%n}',
+      '--pretty=format:{%n  "commit": "%H",%n  "tree": "%T",%n  "subject": "%s",%n  "author": {%n    "name": "%aN",%n    "email": "%aE",%n    "time":"%at"  },%n  "committer": {%n    "name": "%cN",%n    "email": "%cE",%n    "time":"%ct"  }%n}',
```

Comment:
This relates to the previous item... It's potentially an api change (output from something to something else).